R/project.config.R

```diff
@@ -4,62 +4,66 @@
 #' \code{config/global.dcf} that contains various options that can be tweaked
 #' to control runtime behaviour.  The valid options are shown below, and must
 #' be encoded using the \code{DCF} format.
-#' 
+#'
 #' Calling the \code{project.config()} function will display the current project
 #' configuration.
-#' 
-#' 
-#' @return  The current project configuration is displayed. 
-#' 
-#' 
+#'
+#'
+#' @return  The current project configuration is displayed.
+#'
+#'
 #' @details The options that can be configured in the \code{config/global.dcf} are
 #'  shown below
 #' \tabular{ll}{
 #'  \code{data_loading} \tab This can be set to 'on' or 'off'. If data_loading is on,
-#'  the system will load data from both the cache and data directories with 
+#'  the system will load data from both the cache and data directories with
 #'  cache taking precedence in the case of name conflict. \cr
+#'  \code{data_ignore} \tab A comma separated list of files to be ignored when importing
+#'  from the \code{data/} directory. Regular expressions can be used but should be delimited
+#'   (on both sides) by \code{/}. Note that filenames and filepaths should *never* begin with
```

Comment:
I'm not sure this is markdown - the asterix would just get carried through to the man page I think.  

I don't think emphasis is used in man pages.